chromium_src/third_party/blink/renderer/core/execution_context/execution_context.cc

```diff
@@ -176,7 +176,8 @@ scoped_refptr<blink::StaticBitmapImage> BraveSessionCache::PerturbPixels(
 scoped_refptr<blink::StaticBitmapImage>
 BraveSessionCache::PerturbPixelsInternal(
     scoped_refptr<blink::StaticBitmapImage> image_bitmap) {
-  DCHECK(image_bitmap);
+  if (!image_bitmap)
+    return nullptr;
```

Comment:
Could this cause a crash in `canvas_async_blob_creator.cc` where `PerturbPixels` is called and then `image_->PaintImageForCurrentFrame()`?

Reply:
good catch. fixed in https://github.com/brave/brave-core/commit/f0a42f3fe5ee511507ed04cd55c7167516d3cdda